### PR TITLE
Update to fix replication that broke with MySQL 8.4.0 removal of master and slave commands

### PR DESCRIPTION
**Pull request by dlmorri75**
### Description

Fixes and issue with replication that broke with MySQL 8.4.0 removal of depreciated master and slave replication commands.

- #19281

### CODE REVIEW

**Comment by MauricioFauth** [comment]
- Related to #16197

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Thank you for your contribution. 

These changes will break for older versions of MySQL, as phpMyAdmin supports MySQL/MariaDB >= 5.5.

So, it should check if the server is MySQL >= 8.4 to use the new statements, otherwise use the old ones.

File: libraries/classes/Replication.php [hunk 1]
```diff
@@ -95,16 +99,29 @@ public function replicaChangePrimary(
...
-            'MASTER_USER=\'' . $user . '\',' .
-            'MASTER_PASSWORD=\'' . $password . '\',' .
-            'MASTER_LOG_FILE=\'' . $pos['File'] . '\',' .
-            'MASTER_LOG_POS=' . $pos['Position'] . ';',
-            $link
-        );
+        if ($dbi->isMySql() && $dbi->getVersion() >= 80400) {
+            $out = $dbi->tryQuery(
+                'CHANGE REPLICATION SOURCE TO ' .
+                'SOURCE_HOST=\'' . $host . '\',' .
```
Comment (kamil-tekiela) [inline libraries/classes/Replication.php:105]: Shouldn't this use `quoteString()`?

Comment (MauricioFauth) [reply libraries/classes/Replication.php:105]: I thought so, but 5.2 doesn't have `quoteString` and those variables were escaped earlier using `escapeString`.

**Comment by MauricioFauth** [comment]
@dlmorri75 Merged! Thanks!

**Pull request merged**